admin-dev/themes/new-theme/js/pages/order/view/order-product-add-autocomplete.js

```diff
@@ -49,16 +49,16 @@ export default class OrderProductAutocomplete {
   }
 
   search(search) {
-    $.ajax(this.router.generate('admin_products_search', {search_phrase: search}), {
+    $.get(this.router.generate('admin_products_search', {search_phrase: search}), {
```

Comment:
I'm not sure but I think if you use `$.get` the method option becomes redundant and can be removed?

Reply:
It's a fail :trollface:

Reply:
Fixed